src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/boutgifter/domain/BeregningsresultatBoutgifter.kt

```diff
@@ -28,6 +29,13 @@ data class BeregningsresultatForLøpendeMåned(
         fom: LocalDate,
         tom: LocalDate,
     ): BeregningsresultatForLøpendeMåned = this.copy(grunnlag = this.grunnlag.copy(fom = fom, tom = tom))
+
+    fun markerSomDelAvTidligereUtbetaling() = this.copy(delAvTidligereUtbetaling = true)
+
+    fun harUtgiftFørDagensDato(): Boolean =
+        this.grunnlag.utgifter.values
+            .flatten()
+            .any { utgift -> utgift.fom < LocalDate.now() }
```

Comment:
Denne delen kunne fint vært en del av `MarkerSomDelAvTidligereUtebtalingUtils`, men la den her for å mocke resultetet med `LocalDate.now()` i testene. Tar gjerne i mot bedre måter å løse dette på.

Reply:
Det var lurt. Gjør det 👍